CODEOWNERS

```diff
@@ -10,32 +10,83 @@
 # when someone opens a pull request that modifies code that they own.
 # Later matches take precedence.
 
-src/dmd/access.d @MartinNowak
-src/dmd/astbase* @RazvanN7
-src/dmd/astcodegen.d @RazvanN7
-src/dmd/asttypename.d @UplinkCoder
-src/dmd/attrib.d @Geod24
-src/dmd/builtin.d @klickvebot @WalterBright
-src/dmd/cond.d @Geod24
-src/dmd/console.d @CyberShadow
-src/dmd/cppmangle.d @ibuclaw @Geod24
-src/dmd/cparse.d @ibuclaw @WalterBright
-src/dmd/ctfeexpr.d @UplinkCoder
-src/dmd/doc.d @andralex @jacob-carlborg
-src/dmd/dscope.d @Geod24
-src/dmd/hdrgen.d @UplinkCoder @WalterBright
-src/dmd/iasm.d @ibuclaw @WalterBright
-src/dmd/iasmdmd.d @WalterBright
-src/dmd/iasmgcc.d @ibuclaw
-src/dmd/mars.d @MartinNowak @Geod24 @rainers @UplinkCoder @WalterBright
-src/dmd/objc* @jacob-carlborg
-src/dmd/permissivevisitor.d @RazvanN7
-src/dmd/target.d @ibuclaw @MartinNowak
-src/dmd/transitivevisitor.d @RazvanN7
-src/dmd/vcbuild @rainers
+compiler/src/dmd/access.d @MartinNowak
+compiler/src/dmd/astbase* @RazvanN7
+compiler/src/dmd/astcodegen.d @RazvanN7
+compiler/src/dmd/asttypename.d @UplinkCoder
+compiler/src/dmd/attrib.d @Geod24
+compiler/src/dmd/builtin.d @dnadlinger @WalterBright
+compiler/src/dmd/cond.d @Geod24
+compiler/src/dmd/console.d @CyberShadow
+compiler/src/dmd/cppmangle.d @ibuclaw @Geod24
+compiler/src/dmd/cparse.d @ibuclaw @WalterBright
+compiler/src/dmd/ctfeexpr.d @UplinkCoder
+compiler/src/dmd/doc.d @andralex @jacob-carlborg
+compiler/src/dmd/dscope.d @Geod24
+compiler/src/dmd/hdrgen.d @UplinkCoder @WalterBright
+compiler/src/dmd/iasm.d @ibuclaw @WalterBright
+compiler/src/dmd/iasmdmd.d @WalterBright
+compiler/src/dmd/iasmgcc.d @ibuclaw
+compiler/src/dmd/mars.d @MartinNowak @Geod24 @rainers @UplinkCoder @WalterBright
+compiler/src/dmd/objc* @jacob-carlborg
+compiler/src/dmd/permissivevisitor.d @RazvanN7
+compiler/src/dmd/target.d @ibuclaw @MartinNowak
+compiler/src/dmd/transitivevisitor.d @RazvanN7
+compiler/src/dmd/vcbuild @rainers
+
+# D Runtime
+druntime/* @andralex @wilzbach @JackStouffer
+
+druntime/src/checkedint.d @redstar @andralex @JackStouffer
+
+druntime/src/core/atomic.d @WalterBright @ibuclaw @TurkeyMan
+druntime/src/core/attribute.d @jacob-carlborg
+druntime/src/core/bitop.d @schveiguy @tsbockman @Geod24
+druntime/src/core/cpuid.d @WalterBright @ibuclaw @JackStouffer
+druntime/src/core/demangle.d @WalterBright @MartinNowak @rainers @ibuclaw
+druntime/src/core/exception.d @MartinNowak @WalterBright @jmdavis @CyberShadow
+druntime/src/core/internal @MartinNowak @schveiguy
+druntime/src/core/int128.d @WalterBright
+druntime/src/core/math.d @ibuclaw @redstar
+druntime/src/core/runtime.d @MartinNowak @Abscissa
+druntime/src/core/simd.d @WalterBright @MartinNowak
+druntime/src/core/stdc/* @schveiguy @ibuclaw
+druntime/src/core/stdcpp/* @WalterBright @Darredevil @TurkeyMan
```

Comment:
@RazvanN7 : What's the status of @Darredevil ?

Reply:
You can remove him. He has been contributing in at least 4 years.